recipes/nmos-cpp/all/patches/avahi-target.patch

```diff
@@ -0,0 +1,18 @@
+diff --git a/Development/cmake/NmosCppDependencies.cmake b/Development/cmake/NmosCppDependencies.cmake
+index 58bf1e3c..281da6cc 100644
+--- a/Development/cmake/NmosCppDependencies.cmake
++++ b/Development/cmake/NmosCppDependencies.cmake
+@@ -264,7 +264,12 @@ if(${CMAKE_SYSTEM_NAME} STREQUAL "Linux")
+             message(STATUS "Found Avahi version " ${Avahi_VERSION})
+         endif()
+ 
+-        target_link_libraries(DNSSD INTERFACE Avahi::compat-libdns_sd)
++        if(TARGET Avahi::compat-libdns_sd)
++            target_link_libraries(DNSSD INTERFACE Avahi::compat-libdns_sd)
++        else()
++            # unfortunately some versions of the Conan 'avahi' recipe may produce a lowercase namespace
```

Comment:
```suggestion
+            # The upstream does not provide CMake targets, so Conan uses the package name as default target
```

It's wrong to assume Avahi is the correct cmake target as the upstream does not provide cmake files